apis/cluster/v1beta1/internalmembercluster_types.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:scope=Namespaced,categories={fleet},shortName=imc
+// +kubebuilder:resource:scope=Namespaced,categories={fleet-cluster},shortName=imc
```

Comment:
It might be better to specify two categories here: `fleet`, and `fleet-cluster`.

Reply:
can one use selector to select CRDs based on its category?

Reply:
They can use category to select CRs